src/global_distrib/mod_global_distrib_mapping.erl

```diff
@@ -114,6 +115,9 @@ all_domains() ->
 endpoints(Host) ->
     mod_global_distrib_mapping_backend:get_endpoints(Host).
 
+hosts() ->
```

Comment:
what is hosts?

Reply:
These are all the hosts registered in Redis. I'm gonna add a spec here. I called it according to `endpoints/1` function. What more should be changed?